stripe/src/main/java/com/stripe/android/paymentsheet/ui/PrimaryButton.kt

```diff
@@ -97,7 +108,7 @@ internal class PrimaryButton @JvmOverloads constructor(
     }
 
     internal sealed class State {
-        data class Ready(val label: String) : State()
+        data class Ready(val label: String? = null) : State()
```

Comment:
Can you add a small comment that clarifies when `label` will be applied (i.e. when it is non-null)?